sdk/communication/azure-communication-email/azure/communication/email/_email_client.py

```diff
@@ -48,11 +53,10 @@ def __init__(
 
         self._api_version = kwargs.pop("api_version", DEFAULT_VERSION)
 
-        authentication_policy = HMACCredentialsPolicy(endpoint, credential)
+        authentication_policy = get_authentication_policy(endpoint, credential)
```

Comment:
It seems that this `_email_client.py` file (and the async equivalent) is pretty redundant now, as we could easily move these pieces into the `_patch.py` files. Though you could leave that refactor for the next preview if you prefer.

Reply:
Gotcha. Will leave this refactor for the next preview.